db/seeds.rb

```diff
@@ -6,5 +7,8 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 CSV.foreach('./data/ProcurementCard_2017.csv', headers: true, header_converters: :symbol) do |row|
-
+  vendor = Vendor.create(name: row[:merchant_name], state: row[:merchant_state]) unless Vendor.find_by_name(row[:merchant_name])
+  program = Program.create(name: row[:program_area]) unless Program.find_by_name(row[:program_area])
+  purchase = Purchase.create!(transaction_date: row[:transaction_date], payment_date: row[:payment_date], description: row[:item_description], amount: row[:transaction_amount])
+  ProgramVendorPurchase.create(purchase: purchase, program: program, vendor: vendor)
```

Comment:
@iandouglas (@ali eventually) This line in my seed file only creates 29 rows instead of the expected ~50k. Any thoughts on how to fix? I think the correct number of vendors, programs and purchases are being created, so I think this is the problem.

Reply:
Apologies to @ali for being mis-tagged. @alischelereth
@ssciolist have you tried setting your ProgramVendorPurchase model's create using the explicit field names like `.create(purchase_id: purchase.id, program_id: program.id, etc)` ?

Reply:
Yikes, sorry about the mistag. Taking a look now, thank you.

Reply:
Figured it out, thank you so much!